content/posts/why-sre/index.mdx

```diff
@@ -0,0 +1,163 @@
+ <!-- Title, author, date block standard in all blog post files -->
```

Comment:
HTML comments don't work in MDX
This might work: https://docs.microsoft.com/en-us/sql/mdx/comments-mdx-syntax?view=sql-server-ver15

Reply:
@durandom does it have to be mdx?
I'm fine with md. (I just copied what was used for the hello-world)